lib/server.go

```diff
@@ -2439,10 +2442,12 @@ func (srv *Server) _handleBlockBundle(pp *Peer, bundle *MsgDeSoBlockBundle) {
 		// though as we'll just connect all the blocks after the failed one and those blocks will also
 		// gracefully fail.
 		srv._handleBlock(pp, blk, ii == len(bundle.Blocks)-1 /*isLastBlock*/)
-		numLogBlocks := 1000
-		if srv.params.IsPoWBlockHeight(blk.Header.Height) {
-			numLogBlocks = 100
+		numLogBlocks := 100
+		if srv.params.IsPoWBlockHeight(blk.Header.Height) ||
```

Comment:
I think you want to check if it's PoS
```suggestion
		if srv.params.IsPoSBlockHeight(blk.Header.Height) ||
```